README.md

```diff
@@ -94,6 +93,36 @@ Both of these have out-of-tree alternatives in the form of [CSI](https://github.
 
 We remove these to achieve a smaller binary size. They can be removed while remaining conformant because neither affects core Kubernetes functionality. They are also dependent on third-party cloud or data center technologies/services, which may not be available in many K3s' use cases.
 
+Getting Started
+---
+- [Quick Install](https://docs.k3s.io/quick-start)
+- [Achictecture](https://docs.k3s.io/architecture)
+- [FAQ](https://docs.k3s.io/faq)
+- [Contribute](CONTRIBUTING.md)
+
+Community
+---
+- ### Slack
+
+Join [Slack](https://slack.rancher.io/) to chat with K3s developers and other K3s users. Great place to learn and ask questions: [#k3s](https://rancher-users.slack.com/archives/CGGQEHPPW) and [#k3s-contributor](https://rancher-users.slack.com/archives/CGXR87T8B)
```

Comment:
should we start pushing to use the cncf slack instead of rancher users?

Reply:
It's not a hard written requirement, it's good if we can support the CNCF K3s slack channel as we should hang out there, but CNCF has a rule if community does already exists somewhere else we can keep that but still we make sure we follow up in CNCF slack. Asked @manuelbuil if we monitor the https://cloud-native.slack.com/archives/C0196ULKX8S but looks like it's not very active. I'm all for adding it but don't want to introduce overhead. :)